Assets/Scripts/Event.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System.Linq;
+
+public class Event : MonoBehaviour {
+    
+    public Location location;
+    public Item item;
+    public float time;
```

Comment:
because i didn't think of that? the way I got it was just Time which is a float, it's the number of seconds since start. Datetime might work better but I didn't know about it, and that was the first one that came up in the time class that i could find and implement. I think this will work fine since it won't start and stop. i'm not sure of the processing implications , pros/cons of either though.